homeassistant/components/mazda/__init__.py

```diff
@@ -59,6 +65,77 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         _LOGGER.error("Error occurred during Mazda login request: %s", ex)
         raise ConfigEntryNotReady from ex
 
+    async def async_handle_service_call(service_call=None):
+        """Handle a service call."""
+        # Get device entry from device registry
+        dev_reg = device_registry.async_get(hass)
+        device_id = service_call.data.get("device_id")
```

Comment:
Please don't use `dict.get` to access a key that we know exist in the dict. Use `dict[key]`.